AnimationGroup.lua

```diff
@@ -60,28 +60,55 @@ function CreateFrame(...)
     return frame
 end
 
-
 local function OnUpdate(self, elapsed)
     if self.paused then
         return
     end
-
-    self.time = self.time + (self.group.reverse and -elapsed or elapsed)
+    local reverse = self.group.reverse
```

Comment:
```suggestion
    local in_reverse = self.group.reverse
```

That makes it more obvious what the outcome should be: "if in_reverse then UpdateProgress('reverse') end"

I was thinking if maybe renaming `self.group.reverse` to `self.group.direction` would make sense. Changing it from a boolean to `-1` or `1`. That way, you don't need to check its value, instead you would just multiply it with `elapsed`, e.g.: `self.time = self.time + self.group.direction * elapsed`